validation_run.py

```diff
@@ -0,0 +1,182 @@
+
+######## Parameter input ##########
+######## Please comment out unused parameters ############
+
+# ## Tianjin office
+# print("*"*5, 'Tianjin office', '*'*5)
+# EMPLOYEE=906
+# WFH=0
+# CASE_RATE=1/100000
+# DAILY_AVERAGE_CONTACTS=[4,10]
+# DESIGN_ACH=[3,8]
+# SPEAKING_PCT=[.05,.25]
+# NUMBER_DAYS=10
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=7
+
+## Korean call center
+print("*" * 5, "Korean Call Center", "*" * 5)
+EMPLOYEE = 216
+WFH = 0
+CASE_RATE = 1 / 1e5
+DAILY_AVERAGE_CONTACTS = np.asarray([4, 10])
+DESIGN_ACH = [3, 8]
+SPEAKING_PCT = np.asarray([0.55, 0.66])
+SPEAKING_VOL = 70
+NUMBER_DAYS = 14
+PROBABILITY_CONTACT_REMAIN = 0.4
+EMPIRICAL = 76
+
+# ## San Diego VA Office #
+# print("*"*5, 'San Diego VA office', '*'*5)
+# EMPLOYEE=100
+# WFH=0
+# CASE_RATE=1/100000
+# DAILY_AVERAGE_CONTACTS=[4,10]
+# DESIGN_ACH=[3,8]
+# SPEAKING_PCT=[.05,.4]
+# NUMBER_DAYS=8
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=5
+
+# ## singapore conference
+# print("*"*5, 'Singapore conference', '*'*5)
+# EMPLOYEE=111
+# WFH=0
+# CASE_RATE=1/1e5
+# DAILY_AVERAGE_CONTACTS=[6,12]
+# DESIGN_ACH=[3,8]
+# SPEAKING_PCT=[.25,.4]
+# NUMBER_DAYS=3
+# PROBABILITY_CONTACT_REMAIN=0
+# EMPIRICAL=7
+
+# ## SD meat processing Shift 1
+# print("*"*5, 'SD Meat Processing Shfit 1', '*'*5)
+# EMPLOYEE=1744
+# WFH=0
+# CASE_RATE=1/1e5
+# DAILY_AVERAGE_CONTACTS=[4,10]
+# DESIGN_ACH=[2,22]
+# SPEAKING_PCT=[.05,.2]
+# NUMBER_DAYS=14
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=32
+
+# ## SD meat processing shift 2
+# print("*"*5, 'SD Meat Processing Shfit 2', '*'*5)
+# EMPLOYEE=1459
+# WFH=0
+# CASE_RATE=1/1e5
+# DAILY_AVERAGE_CONTACTS=[4,10]
+# DESIGN_ACH=[2,22]
+# SPEAKING_PCT=[.05,.2]
+# NUMBER_DAYS=14
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=6
+
+# ## Henan expressway
+# print("*"*5, 'Henan Expressway', '*'*5)
+# ## to run this case, skip the block below to calculate SAR,  instead enter SAR = [0.2, 0.2]
+# EMPLOYEE=103
+# WFH=0
+# CASE_RATE=1/100000
+# DAILY_AVERAGE_CONTACTS=[2,4]
+# DESIGN_ACH=[3,8]
+# SPEAKING_PCT=[.05,.25]
+# NUMBER_DAYS=13
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=6
+
+# ## MLB with mask protocol and social distancing
+# print("*"*5, 'MLB with mask and social distancing', '*'*5)
+# EMPLOYEE=68
+# WFH=0
+# CASE_RATE=60/1e5
+# DAILY_AVERAGE_CONTACTS=[4,8]
+# DESIGN_ACH=[3,8]
+# SPEAKING_PCT=[.05,.25]
+# NUMBER_DAYS=10
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=20
+
+# ## MLB without mask protocol and social distancing
+# print("*"*5, 'MLB without mask and social distancing', '*'*5)
+# EMPLOYEE=68
+# WFH=0
+# CASE_RATE=60/1e5
+# DAILY_AVERAGE_CONTACTS=[4,10]
+# DESIGN_ACH=[3,8]
+# SPEAKING_PCT=[.05,.25]
+# NUMBER_DAYS=10
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=20
+
+######## simulations ##########
+######## Please note that this takes significant amount of time ############
+
+## calculate SAR
+## skip this for Henan Expressway Outbreak
+## Henan Expressway SAR = np.asarray([0.2, 0.2])
+SAR = np.asarray([])
```

Comment:
This code block may be unnecessary. Can be replaced with the following - 
```
covid_model.get_sar_from_speaking_airflow(design_airflow_ACH=DESIGN_ACH,
             filtration_per_hour=0,
             speaking_percentage=SPEAKING_PCT,
             speaking_volume=SPEAKING_VOL,
             mask_effectiveness=0.0,
         )
```
Please check if it works correctly.

Reply:
this doesn't work the same as in the single outbreak validation. See this [notebook](https://github.com/oneconcern/pangea/blob/main/stochastic_model_validation_single_cases/notebook/korean.ipynb) for details
Since I used `get_sar_custom` in the validation, the resulted SAR range is bigger than what was in `get_sar_from_speaking_airflow`

```
import covid_workplace_model as covid_model
covid_model.get_sar_from_speaking_airflow(design_airflow_ACH=DESIGN_ACH,
             filtration_per_hour=0,
             speaking_percentage=SPEAKING_PCT,
             speaking_volume=SPEAKING_VOL,
             mask_effectiveness=0.0)

## result
[0.18352368, 0.10534145]
```

Then in the Korean call center validation
```
sar = new_validate.get_sar_custom(OFFICE, MASK_COMP, 'default', SPEAKING_ACT, SPEAKING_PCT, AIRFLOW, DESIGN_ACH)
## result
array([0.08884311, 0.21539795]
```

So if I call the `get_sar_from_speaking_airflow` using the original block, I get four values. And then I used min and max to get the SAR used for the simulation
```
SAR = np.asarray([])
for pct in SPEAKING_PCT:
  SAR= np.append(SAR, new_model.get_sar_from_speaking_airflow(\
    design_airflow_ACH=DESIGN_ACH,
    filtration_per_hour=0,
    speaking_percentage=pct,
    speaking_volume=SPEAKING_VOL,
    mask_effectiveness=0.0,))

SAR = np.sort(SAR)
## result
array([0.08884311, 0.10534145, 0.18352368, 0.21539795])


```